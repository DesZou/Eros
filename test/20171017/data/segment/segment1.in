10 10 10
20004 71514 70650 65729 21851 54477 46112 98224 30801 89931 
2 2 9 75149
1 6 8 74
2 2 7 87405
2 1 2 91279
2 2 5 29370
2 7 9 7355
1 2 6 12
2 3 8 98943
2 1 7 89102
2 2 9 82857
7 8
1 5
4 7
5 9
5 7
8 9
2 4
10 10
4 10
2 5
