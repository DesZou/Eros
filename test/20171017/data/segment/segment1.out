139569.871
397470.690
314463.367
358535.119
237323.986
121211.133
230140.810
89931.000
525605.500
307280.190
